nerfstudio/model_components/renderers.py

```diff
@@ -101,9 +102,12 @@ def combine_rgb(
             comp_rgb = torch.sum(weights * rgb, dim=-2)
             accumulated_weight = torch.sum(weights, dim=-2)
 
+        if background_color == "random":
+            # If background color is random, the predicted color is blended with black (0, 0, 0).
```

Comment:
Using "added" instead of "blended" would be more accurate

Reply:
Updated the comment.